iaso/utils/models/common.py

```diff
@@ -23,3 +27,67 @@ def get_org_unit_parents_ref(field_name, org_unit, parent_source_ref_field_names
         if parent_ref:
             return f"iaso#{parent_ref}"
         return None
+
+
+def check_instance_bulk_gps_push(queryset: QuerySet) -> (bool, Dict[str, List[int]], Dict[str, List[int]]):
```

Comment:
I think I'd have put that in the `validate()` method of a serializer.
It's kinda strange to find a sole validation function in a `models` file.